stable/database/files/nuobackup

```diff
@@ -87,8 +89,17 @@ function execute_backup() {
                --key "$NUODB_BACKUP_KEY/$db_name/${backup_group}/failed" \
                --value "${failed} ${backupset}" --expected-value "${failed}"
          fi
-      fi 
-      exit $retval
+      fi
+      if [ -n "$retry_failed" ] && [ "$type" = "journal" ] && echo "$output" | grep -q "Full or incremental hot copy must be completed before journal hot copy"; then
```

Comment:
I think the portion of the message being handled is a little misleading when seen in isolation, but I understand why you are not handling only the text about journal being suspended (because the same text appears in the other message).  Instead, can we just put comment here that this is related to the journal hotcopy suspension?

Reply:
Will do.